marquez_client/client.py

```diff
@@ -15,40 +15,29 @@
 import requests
 import time
 import uuid
+import logging
 
 from .models import DatasetType, SourceType, JobType
 from marquez_client import errors
-from marquez_client import log
-from marquez_client.constants import (
-    ENABLE_SSL, DEFAULT_HOST, DEFAULT_PORT, DEFAULT_TIMEOUT_MS
-)
+from marquez_client.constants import (DEFAULT_TIMEOUT_MS)
 from marquez_client.version import VERSION
 from six.moves.urllib.parse import quote
 
-_API_PATH = 'api/v1'
+_API_PATH = '/api/v1'
 _USER_AGENT = f'marquez-python/{VERSION}'
 _HEADERS = {'User-Agent': _USER_AGENT}
 
+log = logging.getLogger(__name__)
```

Comment:
We can remove the logger here as a logger is configured via the import:

```
from marquez_client import log
```

Reply:
logging framework is universal.